src/Exception/BadRequestHttpException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException as BaseBadRequestHttpException;
+
+/**
+ * This exception is thrown when there is an issue with the request.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class BadRequestHttpException extends BaseBadRequestHttpException
```

Comment:
Why did we have to introduce a new exception for this? Are we intentionally only handling the Contao exceptions in our listener, but "ignoring" regular Symfony exceptions? That would mean if my extension uses a third-party bundle that throws such an exception, we would not pretty-render it?